code/game/objects/effects/portals.dm

```diff
@@ -50,6 +50,17 @@
 	creator = null
 	return ..()
 
+/obj/effect/portal/proc/blockExiles(atom/movable/AM)

```

Comment:
This check is bad - players can simply go into a locker or something else to avoid this